schema.graphql

```diff
@@ -75,8 +75,32 @@ type ClaimableReward @entity {
 type ReferrerReward @entity {
 	id: ID!
 	blockHeight: BigInt!
-	referrer: String! @index
-	referree: String!
+	referral: String! @index
+	referrer: String!
+	timestamp: Int!
+	amount: String!
+}
+
+type SetReferral @entity {
```

Comment:
All new added entities are not used in the mapping files - then, I guess, there is no need adding the here